media/apps.html

```diff
@@ -2,23 +2,74 @@
-      <h1>Media application development</h1>
+      <h1>Device interoperability</h1>
       <p>
-        The Web platform is under constant evolution. Device implementations and application developers need to share a common understanding of the functionalities that the platform offers at any given point in time, and of recommended guidance to use these powerful features.
+        Televisions, set-top boxes and other media devices typically ship with hardware architectures that are both extremely powerful to decode and render audio and video, and fairly constrained when it comes to running &mdash;and updating&mdash; applications. Web browser code bases need to be ported to these devices, and integrators need to answer a number of practical questions to adapt the code to practical device constraints. That process also usually happens early in the production cycle. As such, final products may run a code base release that is a few years old with varying degrees of APIs support and quality of implementations. In constrast, personal computers and mobile devices are tailored to run applications and follow the <a href="https://www.w3.org/2001/tag/doc/evergreen-web/">constant evolution</a> of the Web platform, with always up-to-date Web browsers. This creates interoperability issues within media devices as well as between media devices and other types of devices.
+      </p>
+      <p>
+        Media device manufacturers and integrators need to share a common understanding of the functionalities that their platforms need to offer at any given point in time, and of recommended guidance to implement these features, given hardware constraints, to guarantee a suitable user experience. Application developers need ways to identify specific features and constraints of the platform on which their application runs, and mechanisms to measure code performance.
       </p>
     </header>
     <main>
+      <section class="featureset well-deployed">
+        <h2>Well-deployed technologies</h2>
+
+        <div data-feature="Measuring the Quality of Experience">
+          <p>The <a href="https://www.w3.org/webperf/">Web Performance Working Group</a> developed a number of specifications that expose timing hooks to Web applications, to analyze the time spent doing various tasks. Although not specifically targeted at media scenarios, these specifications make it possible to monitor the performance of running applications. <a data-featureid="hr-time">High-Resolution Time</a> exposes a monotonic sub-millisecond resolution clock to Web applications so that they can precisely measure time elapsed between two events. <a data-featureid="performance-timeline">Performance Timeline</a> defines a unified interface to store and retrieve performance metric data. Individual performance metric interfaces are enumerated in the <a data-featureid="timing-entrytypes-registry">Timing Entry Names Registry</a> and defined in separate specifications. Well-deployed interfaces include:</p>
```

Comment:
I haven't been through and checked, but are all these well-deployed techonolgies part of the latest Web Media API?

Reply:
They are not for the time being, but I see that [@JohnRiv is proposing to review these specifications](https://github.com/w3c/webmediaapi/issues/244#issuecomment-581468897)